examples/player/simid_player.js

```diff
@@ -452,4 +453,16 @@ class SimidPlayer {
     }
     this.simidProtocol.resolve(incomingMessage, mediaState);
   }
-}
+
+  onReceiveCreativeLog(incomingMessage) {
+    const logMessage = incomingMessage.args['message']
```

Comment:
stay consistent here and use incomingMessage.args.message;

Reply:
I stay consistent within the files. If you look at for example like 438 in this file, it uses this format